src/services/delayModuleService.ts

```diff
@@ -0,0 +1,102 @@
+import { StaticJsonRpcProvider } from '@ethersproject/providers'
+import { Contract, utils } from 'ethers'
+import { MetaTransaction } from 'ethers-multisend'
+
+import delayModuleABI from '../contracts/abis/Delay.json'
+
+const DelayModule = new utils.Interface(delayModuleABI)
+
+export type DelayContext = {
+  delayModuleAddress: string
+  moduleFactoryAddress: string
+  recoveryCooldown: number
+  recoveryExpiration: number
+}
+
+export const getDelayAddress = async (
```

Comment:
The delay module address should be calculed and returned by the getWalletInfos method

Reply:
done